packages/iframe-coordinator/src/elements/frame-router.ts

```diff
@@ -68,6 +78,9 @@ export default class FrameRouterElement extends HTMLElement {
    *
    * @param clients The map of registrations for the available clients.
    * @param envData Information about the host environment.
+   *
+   * @deprecated Use the new {@configureClients} method instead
```

Comment:
@MattCheely in a previous comment you mentioned that we can deprecate `setupFrames()`. I added a new method `configureClients()` that can replace `setupFrames()`. Is this what you were thinking? Or were you thinking that we would remove the public method all together in favor of using the property?

Reply:
That is a good question. 🤔 I would say that for now, we can just deprecate `setupFrames()` without adding a replacement function. We'll announce the deprecation and get feedback from consumers to see if moving to a property causes any problems for people, and then create the new function if it does. That way, if the property is good enough on it's own there will only be one way to do it. That's usually preferable.